hudi-common/src/main/java/org/apache/hudi/common/util/NumericUtils.java

```diff
@@ -31,4 +38,27 @@ public static String humanReadableByteCount(double bytes) {
     String pre = "KMGTPE".charAt(exp - 1) + "";
     return String.format("%.1f %sB", bytes / Math.pow(1024, exp), pre);
   }
+
+  public static long getMessageDigestHash(final String algorithmName, final String string) {
+    MessageDigest md;
+    try {
+      md = MessageDigest.getInstance(algorithmName);
+    } catch (NoSuchAlgorithmException e) {
+      throw new HoodieException(e);
+    }
+    return asLong(Objects.requireNonNull(md).digest(string.getBytes(StandardCharsets.UTF_8)));
+  }
+
+  public static long asLong(byte[] bytes) {
+    ValidationUtils.checkState(bytes.length >= 8, "HashCode#asLong() requires >= 8 bytes.");
+    return padToLong(bytes);
+  }
+
+  public static long padToLong(byte[] bytes) {
+    long retVal = (bytes[0] & 0xFF);
```

Comment:
do we need & 0xFF ?

Reply:
@s-sanjay do you see a specific problem? may be an example could help?

Reply:
@vinothchandar this is a nitpick comment. there is nothing wrong in and with 0xFF but it is not really needed right ?
I maybe wrong here but my understanding is 
`long retVal = bytes[0]`
and 
`long retVal = (bytes[0] & 0xFF)` are the same

Reply:
>long retVal = (bytes[0] & 0xFF) are the same
hmm yeah.. you are right..